tests/e2e/tests/templates/cart-template.block_theme.spec.ts

```diff
@@ -37,13 +36,14 @@ test.describe( 'Test the cart template', async () => {
 		await editor.page.getByRole( 'button', { name: /Pages/i } ).click();
 		await editor.page.getByRole( 'button', { name: /Cart/i } ).click();
 		await editorUtils.enterEditMode();
+		await editorUtils.closeWelcomeGuideModal();
 		await expect(
-			editor.canvas.locator( 'h2:has-text("Cart")' ).first()
+			editor.canvas.locator( 'h1:has-text("Cart")' ).first()
 		).toBeVisible();
 		await editor.openDocumentSettingsSidebar();
-		await page.getByRole( 'button', { name: 'Edit template' } ).click();
+		await page.getByLabel( 'Template options' ).click();
```

Comment:
We should click the `Edit template` option after clicking on the `template options` label.